img2dataset/blurrer.py

```diff
@@ -0,0 +1,97 @@
+"""blurrer module to blur parts of the image"""
+
+from typing import Optional
+
+import numpy as np
+
+import albumentations as A
+
+
+VALID_BBOX_FORMATS = ["albumentations", "coco"]
```

Comment:
can you pick one and translate to the other if needed ?

Reply:
The idea was that this depends on the input dataset. See my comment below for details.